test/example/c-api-kitchen-sink.c

```diff
@@ -270,6 +270,30 @@ void test_types() {
   BinaryenTypeExpand(dataref, &valueType);
   assert(valueType == dataref);
 
+  BinaryenType stringref = BinaryenTypeStringref();
+  printf("BinaryenTypeStringref: (ptr)\n", stringref);
+  assert(BinaryenTypeArity(stringref) == 1);
+  BinaryenTypeExpand(stringref, &valueType);
+  assert(valueType == stringref);
+
+  BinaryenType stringview_wtf8_ = BinaryenTypeStringviewWtf8();
```

Comment:
Note: The `_` is there to substitute `ref` so these don't conflict with the heap types of the same name that another PR adds tests for below.